packages/stateManager/index.ts

```diff
@@ -615,8 +627,45 @@ export class StateManager {
       this.keycards,
     ];
     const stream = this.client.createEventStream();
+    const publicClient = createPublicClient({
+      chain: hardhat,
+      transport: http(),
+    });
+
+    const count = (await publicClient.readContract({
```

Comment:
_getRelayCount_ and _getRelayCount_ contract calls are not erroring out but it returns empty. Shouldn't it return my locally running address?

Reply:
my deployment addresses seem to all match too

Reply:
By default the shop has zero relays. The function to to do this is called `addRelay`, and it takes the shop and the relay NFT id.

https://github.com/masslbs/pystoretest/blob/a9307bb29057fad4f5053ab2202f1cef8f3361b5/client.py#L296

for MVP i'd do this together with the shop mint

i introduced the relay nft id a while ago but we were only using it for the new enroll. It's bundled up in the RelayEndpoint type:
https://github.com/masslbs/Tennessine/blob/90c10f0f0baa6c0ae5a01283ea4a26741ce3f437/packages/client/src/index.ts#L28